### ISSUE TITLE
Make sure CWTS is on the same page in Rust FxiOS

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#4336]
This is the new experiment in FxA that puts CWTS on the same page.

See 
https://mozilla.slack.com/archives/C4D36CAJW/p1581460669175200?thread_ts=1581457529.172900&cid=C4D36CAJW for details.

When we get the Rust FxiOS component working, then we should make sure we use the new design

<img src=https://user-images.githubusercontent.com/128755/74283975-e66d3480-4cf0-11ea-97b9-25afc0a0b526.png width=300 />

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-1240)
┆Issue Number: FXA-1240

**Comment by data-sync-user** [comment]
➤ Janet Dragojevic commented:

🚩 Flag added

Blocked on some ios work, and not a blocker for the whole epic.

**Comment by data-sync-user** [comment]
➤ Vlad Filippov commented:

This is blocked on https://github.com/mozilla/fxa/issues/4159, I looked at the fxa bit of this and it will a bit of refactoring on the FxA side. We can block on that for now.

**Comment by data-sync-user** [comment]
➤ Janet Dragojevic commented:

Confirmed with Wil, that FxA will triage this into their sprint work.  Moving to the FxA board.  

FYI - this is not considered a release blocker for the iOS version 24 (~Mar 20, 2020), but would be nice to have a consistent sign-in experience.  Recommend connecting with iOS PM, Ashley for product needs.

**Comment by data-sync-user** [comment]
➤ Wil Clouser commented:

We're triaging this, but we aren't sure what to do with it.  Is there a specific ask here?  What design are you seeing?

**Comment by rfk** [comment]
We were seeing the "old" CWTS experience with the options presented on a separate page, rather than inline with the password field. I expect this issue will be resolved by the recent work to graduate the new CWTS experience from "experiment" to "on by default".

**Comment by data-sync-user** [comment]
➤ Chelsea Lewis commented:

Let's confirm when 162 goes out that this has been resolved [~vbudhram@mozilla.com]

**Comment by vladikoff** [comment]
> We were seeing the "old" CWTS experience with the options presented on a separate page, rather than inline with the password field. I expect this issue will be resolved by the recent work to graduate the new CWTS experience from "experiment" to "on by default".

@rfk the new screen is only for `sync users and browsers with multiservice`, so we might have to tweak this still for the OAuth flow

**Comment by data-sync-user** [comment]
➤ Wil Clouser commented:

162 is live.  Are you able to confirm this is fixed?

**Comment by data-sync-user** [comment]
➤ Wil Clouser commented:

Optimistically closing this.  Please let us know if it's not resolved.
